spec/vsphere_vm_clone_spec.rb

```diff
@@ -24,5 +24,10 @@
       expect(subject).to receive(:vim_connection).and_raise ArgumentError
       expect { subject.run }.to raise_error ArgumentError
     end
+
+    it 'tests the testable test' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.